apps/src/sites/hourofcode.com/pages/public/events/index.js

```diff
@@ -257,8 +257,14 @@ function validateFields() {
     } else {
       $('#school-name-error').hide();
     }
+  }
 
-    if ($('#hoc-event-location').val() === '') {
+  if (
+    ['out_of_school', 'after_school'].includes($('#hoc-event-type').val()) ||
+    ($('#hoc-event-type').val() === 'in_school' &&
```

Comment:
It looks like we also use this logic above (to distinguish the 'in school' events where we still need a location). Would we want to create some sort of reusable variable for this?